examples/express-example/README.md

```diff
@@ -0,0 +1,31 @@
+# Express Example.
```

Comment:
Awesome examples, but I'm not sure I want them here. Examples for Enmap are in the gitbook, which isn't currently synced with github but I can do that for the examples to be added there. 

The problem I see with having them in the code is that they're... well, in the code, in node_modules, so it's more space taken by the module, a larger footprint for the file.

Please remove these examples and I'll add a space for them on the gitbook docs and sync it to github so you can add them there.

Reply:
Resolved, moving them to gitbook.

Reply:
Resolved, moved to gitbook.